jjb/device/device-bacnet.yaml

```diff
@@ -19,5 +19,7 @@
       - '{project-name}-{stream}-merge-java'
       - '{project-name}-{stream}-verify-docker'
       - '{project-name}-{stream}-merge-docker'
-      - '{project-name}-{stream}-release-version-java-daily-no-sonar'
+      - '{project-name} github-maven-jobs':
+          triggers:
+            - timed: 'H 11 * * *'
```

Comment:
"triggers" is not a variable you have to use an actual variable here to override the trigger. Hmm now that I think about it I'm not sure how we can override for  github maybe we have to introduce a new variable.

and i'm not sure if it works to set something underneath a job template  (maybe it does, worth a try)

Reply:
I'm removing this now that we have it parameterized in global-jjb.

Reply:
@zxiiro on your comment about doing an override under a job template. I know that it works, Open-O used to and ONAP does use that pattern extensively.